sm-core/src/main/java/com/salemanager/shop/exception/GenericRuntimeException.java

```diff
@@ -0,0 +1,37 @@
+package com.salemanager.shop.exception;
+
+import lombok.Getter;
+
+public class GenericRuntimeException extends RuntimeException{
+    @Getter
+    private String errorCode;
+    @Getter

```

Comment:
We can have a common @Getter annotation on top of class

Reply:
Thank you. Fixed.